ios/RNSalesforceChat.m

```diff
@@ -136,6 +164,43 @@ - (NSDictionary *)constantsToExport
   return @[ChatSessionEnd, ChatSessionStateChanged];
 }
 
+RCT_EXPORT_METHOD(setupChatColorIOS:(double)redValue greenValue:(double)greenValue blueValue:(double)blueValue alphaValue:(double)alphaValue colorToken:(NSString *)colorToken)
+{
+    SCAppearanceConfiguration *appearance = [SCServiceCloud sharedInstance].appearanceConfiguration;
+    UIColor *color = [UIColor colorWithRed:(float)redValue/255.0f green:(float)greenValue/255.0f blue:(float)blueValue/255.0f alpha:(float) alphaValue];
+    
+    int colorIndex = (int)[colorTokens indexOfObject:colorToken];
+        
+    switch (colorIndex) {
```

Comment:
Maps received color tokens with the [ones](https://developer.salesforce.com/docs/atlas.en-us.noversion.service_sdk_ios.meta/service_sdk_ios/customize_colors.htm) defined on the Salesforce SDK. Those are only the ones that the chat can use.